smart-contract-verifier/src/compiler/compilers.rs

```diff
@@ -67,13 +72,16 @@ where
             Err(FetchError::NotFound(version)) => return Err(Error::VersionNotFound(version)),
             res => res?,
         };
+
         let output = {
-            let _timer = metrics::COMPILE_TIME.start_timer();
             let span = tracing::debug_span!(
                 "compile contract with ethers-solc",
                 ver = compiler_version.to_string()
             );
-            let _guard = span.enter();
+            let _span_guard = span.enter();
```

Comment:
let's add another metric which will count how long are we waiting for the queue

Reply:
and another for how many waiters are there (similar as `COMPILES_IN_FLIGHT`, make it `COMPILES_IN_QUEUE`)

Reply:
4 metrics for compilation :)